File: chicky/config/default.js
module.exports = {
  // Server configuration - these are the only required local settings
  port: process.env.PORT || 3000,
  vpsUrl: process.env.REMOTE_SERVER,
  authToken: process.env.AUTH_TOKEN,
  
  // Hardware configuration - these will be provided by the server
  servo1_pin: null,
  servo2_pin: null,
  
  // Operation settings - these will be provided by the server
  debug: false,
  allowed_start_hour: null,
  allowed_end_hour: null,
  timezone_offset: null
};

File: chicky/config/default.py
"""
Default configuration for the Chickenfeed Pi component
"""
import os
import time
from typing import Dict, Any

# Configuration dictionary
config: Dict[str, Any] = {
    # Server configuration - these are the only required local settings
    "port": int(os.getenv("PORT", "3000")),
    "remote_server": os.getenv("REMOTE_SERVER"),
    "auth_token": os.getenv("AUTH_TOKEN"),
    
    # Hardware configuration - these will be provided by the server
    "servo_pin": None,
    
    # Operation settings - these will be provided by the server
    "allowed_start_hour": None,
    "allowed_end_hour": None,
    "timezone_offset": None,
    
    # Internal settings
    "start_time": time.time()
} 